docs: Add documentation for user-provided API token secret (#1054)
keptn/keptn.github.io: content/docs/0.13.x/operate/advanced_install_options/index.md

      control-plane.ingress.pathType=<YOUR_PATH_TYPE>,  
      control-plane.ingress.tls=<YOUR_TLS>
```

### Install Keptn using a user-provided API token

> If you want to provide your own API token for Keptn to use, you can do that by providing the secret name

Reviewer: ```suggestion
You can provide your own API token for Keptn to use by setting the secret name
```
